client-js/app/elements/dir-item.html

```diff
@@ -0,0 +1,30 @@
+
+<dom-module id='dir-item'>
+  <template>
+    <paper-icon-item><iron-icon icon="{{dir_icon}}" item-icon></iron-icon><content></content></paper-icon-item>
+
+
+  </template>
+
+  <script>
+    Polymer({
+        is:"dir-item",
+        properties: {
+            dir_name: {type: String},
+            prop: {type: String},
+            dir_icon:{
+                type: String,
+                computed: 'assignIcon(prop)'
+            }
+        },
+        assignIcon: function(entry_type){
+            if (entry_type == "folder"){
```

Comment:
I like having a space before the opening curly bracket, both on control structures and function defs. This is also the recommendation in the google javascript style guide: http://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting